2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,13 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  // TODO
+  let temperature = [];
+  for (city of cities) {
+    let temp = temperatureService(city);
+    temperature.push("The temperature in", city, " is ", temp, " degrees");
```

Comment:
The logic is good, the only issue is the way formatting. The idea of pushing the content of this line is excellent, but it needs to be done as a single entry. The way it is currently formatted, js thinks you are pushing 05 differents entries.
Hint: Use backtick here
```
temperature.push(`The temperature in ${city} is ${temp} degrees`);
```